app/assets/javascripts/oxalis/view/settings/user_settings_view.js

```diff
@@ -199,6 +201,9 @@ class UserSettingsView extends PureComponent {
         this.props.tracing.activeNodeId != null ? this.props.tracing.activeNodeId : "";
       const activeTreeId =
         this.props.tracing.activeTreeId != null ? this.props.tracing.activeTreeId : "";
+      const activeNodeRadius =
+        Utils.toNullable(getActiveNode(this.props.tracing).map(activeNode => activeNode.radius)) ||
+        0;
```

Comment:
Good catch, I had another default value with another data type there before and then I forgot ^.-